examples/foomedical/src/pages/ChatsPage.tsx

```diff
@@ -0,0 +1,63 @@
+import { Divider, Stack, Table, Title } from '@mantine/core';
+import { formatGivenName, formatFamilyName } from '@medplum/core';
+import { Patient, Practitioner, HumanName } from '@medplum/fhirtypes';
+import { Document, useMedplum, useMedplumProfile } from '@medplum/react';
+import { useEffect, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { Loading } from '../components/Loading';
+import classes from './ChatsPage.module.css';
+
+export function Chats(): JSX.Element {
+  const medplum = useMedplum();
+  const profile = useMedplumProfile() as Patient;
+  const navigate = useNavigate();
+  const [practitioners, setPractitioners] = useState<Practitioner[]>();
+
+  useEffect(() => {
+    medplum
+      .graphql(
+        `
+        {
+          PractitionerList {
+            id
+            name {
+              given
+              family
+            }
+          }
+        }
+      `
+      )
+      .then((value) => setPractitioners(value.data.PractitionerList as Practitioner[]))
+      .catch((err) => console.error(err));
```

Comment:
Suggestion: It seems that the newer examples are rendering a notification toaster instead of `console.error`. To make it work, you'd need to change [those lines](https://github.com/medplum/medplum/pull/4770/files/776e004fb853100ef7519d3ed178c1362ae21e0c#diff-0bb21ea90801664475b90b26026a04edeadc31cc82ed96f38e52bc36535a5345) that I also did on #4770.


```suggestion
      .catch((err) => {
        showNotification({
          color: 'red',
          icon: <IconCircleOff />,
          title: 'Error',
          message: normalizeErrorString(err),
        });
      });
```

That was an impression of mine, not sure if it's indeed a recommended/official approach or something like that.

Reply:
Thanks @pamellal, I'm updating it all calls